### DISCUSSION TITLE
Install via Homebrew

### DISCUSSION THREAD

**Discussion by muescha** [CommandPost/CommandPost#3257] [General]
It would be nice, if CommandPost can be installed via Homebrew.

I see there are 2 options possible:
- (A) create a cask receipt in https://github.com/Homebrew/homebrew-cask
- (B) create an own Tap for CommandPost

A: Receipt in homebrew-cask:
pro: find the receipt without special setup with `brew search commandpost` / `brew install commandpost`
con: releases need to approved by the maintainers

B: Tap repo
pro: release updates to the cask receipt can be included in the CI workflows
con: user need to know the brew command for the cask: `brew install commandpost/tap-name/commandpost`

**Comment by latenitefilms** [comment]
Apologies for the delayed reply! I'll look into this when I get a chance, however, feel free to make a pull request if it's something you can do yourself!

**Comment by IlIllIl** [comment]
> A: Receipt in homebrew-cask: pro: find the receipt without special setup with `brew search commandpost` / `brew install commandpost` con: releases need to approved by the maintainers

Looks like it's already done by someone. You can try it.
